source/includes/_authentication.md

```diff
@@ -33,3 +33,17 @@ If you need to test that you have a valid API key use the following endpoint:
 ### Response
 
 Successful authentication with return a `200` response, otherwise a `401` will be returned when supplied with an invalid API key.
+
+> 200
+
+```json
+{
+    "data": {
+        "id": "de9559a5dsjhsj74da6afb75a4dsds1e2769b",
```

Comment:
that's not a real one right? 👀

Reply:
...um, no 😂  ....but it's close, so will generate a random one....